uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_request.scss

```diff
@@ -0,0 +1,68 @@
+.request-attachment {
+
+  &-name {
+    color: $gray-darker;
+    font-weight: 600;
```

Comment:
i can improve this in the PRs for the next email templates (TOIL, Sickness), on a tight deadline to merge this now